packages/manager/modules/overthebox/src/overthebox/details/translations/Messages_fr_FR.json

```
@@ -78,15 +77,20 @@
   "overTheBox_release_channel_v0_9": "Version v0.9",
   "overTheBox_release_channel_v1_0": "Version v1.0",
   "overTheBox_version_system": "Version du système d'exploitation",
-  "overTheBox_last_seen": "Dernier accès du matériel depuis les serveurs de provisionning",
+  "overTheBox_last_seen": "Dernière connexion du périphérique",
   "overTheBox_last_seen_few_seconds": "Il y a quelques secondes",
   "overTheBox_last_seen_minutes_ago": "Il y a {{lastSeenDiff}} environ",
   "overTheBox_reverse_dns": "Reverse DNS",
   "overTheBox_detail_link_guide": "Guides",
   "overthebox_detail_links": "Liens",
   "overTheBox_service_id": "Identifiant du service :",
-  "overTheBox_available_update": "Une mise à jour est disponible pour votre appareil.",
+  "overTheBox_available_update": "Une mise à jour est disponible pour votre périphérique.",
   "overTheBox_authorize_autoupgrade": "Autoriser les mises à jour automatiques la nuit",
   "overTheBox_change_auto_upgrade_success": "Le changement des mises automatiques la nuit a été effectué avec succès.",
-  "overTheBox_change_auto_upgrade_error": "Nous n'avons pas pu effectuer le changement des mises automatiques la nuit : {{errorMessage}}"
+  "overTheBox_change_auto_upgrade_error": "Nous n'avons pas pu effectuer le changement des mises automatiques la nuit : {{errorMessage}}",
```

Some words are missing, aren't they ?
```suggestion
  "overTheBox_change_auto_upgrade_error": "Nous n'avons pas pu effectuer le changement des mises à jour automatiques la nuit : {{errorMessage}}",
```

---

I think you have forgot this comment.